### PR TITLE
style: small UI fixes

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/6053/desktop-snackbars-don-t-show-on-zero-chat-screen
https://app.clubhouse.io/peerio/story/7247/desktop-don-t-show-desktop-notification-when-invited-contact-joins-if-app-is-in-foreground
https://app.clubhouse.io/peerio/story/5223/desktop-unread-chats-should-be-white
 
#### Testing instructions  
Several small UI fixes
* snackbars show now on chat zero state
  * were already working on files zero state
* "invite accepted" desktop notification should not show when the app is focused
* in ChatList, unread chat names are solid white, no opacity

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/helpers/notifications.js [hunk 1]
```diff
@@ -206,16 +206,18 @@ class InviteAcceptedNotification {
...
-        if (!username) return;
-        const contact = contactStore.getContact(username);
-
-        this.postDesktopNotification(
-            t('notification_inviteAcceptedTitle'),
-            t('notification_inviteAcceptedBody', {
-                firstName: contact.firstName || '', username
-            })
-        );
+        if (!appState.isFocused) {
```
Comment (anri-asaturov) [inline src/helpers/notifications.js]: Try not to increase nesting when not needed.
You could do:
`if (!appState.isFocused) return;`

**Comment by anri-asaturov** [comment]
ready to test

**Comment by ebilg** [comment]
something seems to be wrong with snackbars

screen shot 2018-07-04 at 4 49 17 pm

**Comment by suprko** [comment]
@ebilg 🤦‍♀️ fixed now

**Comment by ebilg** [comment]
can be merged

**Pull request merged**